api/ssi-bridge/src/test/mocks/bitmap.ts

```diff
@@ -0,0 +1,7 @@
+import { Bitmap } from '@iota/is-shared-modules/lib/web/models/types/identity';
```

Comment:
With the new shared-modules a simplified import should be possible:
```suggestion
import { Bitmap } from '@iota/is-shared-modules';
```